tagger.py

```diff
@@ -0,0 +1,60 @@
+import subprocess
+import shlex
+import csv
+import sys  
+
+RUN_TAGGER_CMD = "java -XX:ParallelGCThreads=2 -Xmx500m -jar ark-tweet-nlp-0.3.2.jar"
```

Comment:
Its CMU POS Tagger for Tweets. I have added it in the comments. I will comment the code later on since its still undergoing revisions